distribution/docs/src/main/resources/_contents/_draft-platform-contents/country-code-converter-contents.adoc

```diff
@@ -0,0 +1,21 @@
+==== Country Code Converter
```

Comment:
Since this is an internal API, should we move this documentation to a README.adoc somewhere in the  platform/country-converter area?